packager/mpd/base/mpd_utils.cc

```diff
@@ -417,6 +439,11 @@ void AddContentProtectionElementsHelperTemplated(
     } else {
       drm_content_protection.scheme_id_uri = "urn:uuid:" + entry.uuid();
       if (!entry.pssh().empty()) {
+        if(entry.uuid() == kPlayReadyUUID && protected_content.allow_mspr_pro()) {
+          drm_content_protection.subelements.push_back(
+              GenerateMsprProElement(entry.pssh()));
+          drm_content_protection.value = kContentProtectionValueMSPR20;
+        }
```

Comment:
Prefer moving it after line 448 so the element is generated after cenc:pssh as in PlayReady spec examples.

Reply:
Done